app/common/index.js

```diff
@@ -20,16 +20,26 @@ export { default as NetworkImage } from './src/image/NetworkImage';
 export { default as FullPageLoading } from './src/loaders/FullPageLoading';
 
 export { default as PriceMarker } from './src/map/PriceMarker';
+export { default as HotelCard } from './src/map/HotelCard';
+export { default as Address } from './src/map/Address';
```

Comment:
I do not think this should be in this shared package "just in case". I do not think there is high chance that we'll reuse these components. It's absolutely fine to create local components and use this only for true "shared" things. I am actually considering rename `common` to `shared` (like in React). I'll leave it here for now. I just want to mention it... 🙂

Reply:
good point, I'm actually going to change it today :)